src/common/items/CustomItem.ts

```diff
@@ -76,13 +77,20 @@ export class CustomItem<T extends ObjectShape> {
    */
   private options: CustomItemOptions<T>;
 
+  /**
+   * CustomModelData to assign and check for this item.
+   */
+  private customModelData: number;
+
   /**
    * Type of data associated with this item.
    */
   private dataType: DataType<T>;
 
   constructor(options: CustomItemOptions<T>) {
     this.options = options;
+    this.customModelData =
+      options.customModel !== false ? options.id : 100000 + options.id;
```

Comment:
Is this +100000 the way to go, or should we consider other ways?

Reply:
Discussed this on Discord, this is the way to go.